docs/docs/models-usage.md

```diff
@@ -501,6 +501,30 @@ User.findAll({ include: [{ model: Tool, as: 'Instruments' }] }).then(function(us
 })
 ```
 
+As alternative you can specify only alias.
```

Comment:
`You can also include by alias name by specifying a string that matches the association alias:`